### PR TITLE
Update scaling SOPs for 2.x

### PR DESCRIPTION
**Pull request by CathalOConnorRH**
Update scaling SOPs for 2.x.

### CODE REVIEW

**Comment by CathalOConnorRH** [comment]
@ciaranRoche Would you be able to review this when you get a chance please?

File: sops/2.x/sso-scaling.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+```
+
+### RHSSOUser
+#### Scalability:
+- You can scale this component **horizontally** by adding more PODs
+```
+oc patch Keycloak rhssouser -n <ns-prefix>-user-sso --patch '{"spec": {"instances": <number-of-replicas> }}' --type=merge
+```
+
+### Relational Database
```
Comment (ciaranRoche) [inline sops/2.x/sso-scaling.md]: Is this needed, now that we have Postgres backed on AWS?

**Review by ciaranRoche — COMMENTED** [review]
Looks good, just one comment in line

**Review by ciaranRoche — APPROVED** [review]
looks good :+1:

**Comment by CathalOConnorRH** [comment]
@david-martin Would you be able to merge this please?

**Comment by david-martin** [comment]
@CathalOConnorRH Merged.
I don't think this repo is used by SRE.
If the intention is to have these SOPs as an option for SRE, they would better fit in the help repo https://github.com/RHCloudServices/integreatly-help/tree/master/sops/2.x

**Comment by CathalOConnorRH** [comment]
@david-martin This repo was suggested by Wei when opening the ticket. I've also added an SRE based SOP into the integreatly-help repo. https://github.com/RHCloudServices/integreatly-help/pull/474

**Pull request merged**